pkgs/by-name/te/termineter/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "termineter";
+  version = "1.0.4";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "rsmusllp";
+    repo = "termineter";
+    rev = "v${version}";
+    hash = "sha256-WZG5njKZxlhokRvghphH458e4b3i+ZWCn6zYtBGjvyM=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    crcelk
+    pluginbase
+    pyasn1
+    pyserial
+    smoke-zephyr
+    tabulate
+    termcolor
+    setuptools
```

Comment:
If the change indicated above affects the building process, try adding setuptools to `propagatedBuildInputs`.